playbooks/roles/deploy-osh/templates/libvirt.yaml.j2

```diff
@@ -6,4 +6,8 @@ conf:
       user: {{ ses_cluster_configuration['cinder']['rbd_store_user'] }}
       keyring: {{ suse_ses_cinder_pool_key | default(ceph_admin_keyring_key) }}
       secret_uuid: {{ libvirt_ceph_cinder_secret_uuid }}
-{{ lookup('template','files/common-images-overrides.yml') | from_yaml | to_nice_yaml(indent=2) }}
+images:
+  tags:
+    # libvirt: "{{ suse_osh_registry_location }}/openstackhelm/libvirt:{{ suse_infra_image_version }}"
+    # Using static tag name as currently it is saved incorrectly in dockerhub
```

Comment:
maybe nit but could we point here to the upstream fix for this: https://review.openstack.org/#/c/648657/